modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
 
 After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-To create a API instance:
+To create an API instance:
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+. Select *API Administration* > *Manage API* > *Create a new API*.
```

Comment:
Here you are changing the flow from "Manage API from Exchange" to "Create New API" is this change intended?

- "Manage API from Exchange": User is able to search and select an asset published in Exchange
- "Create New API": Will allow user to upload an asset to Exchange and Manage such assets in one step.